melodic/distribution.yaml

```diff
@@ -4595,6 +4595,21 @@ repositories:
       url: https://github.com/fkanehiro/hrpsys-base.git
       version: master
     status: maintained
+  http_relay:
```

Comment:
I agree this functionality isn't directly ROS-related. But it is robot-related. A few times we had devices which could not access outer networks directly.

I searched the whole apt and pip, but i haven't found any simple package for http proxying (except for big beefy frameworks).

And as it wasn't the first time we needed to run a http relay on the robots, I felt this would be a good package candidate.

Our latest use-case is directly embedding the proxy into sensor ROS driver.

I think this package follows the naming REP. The name clearly says what it does. Name collisions don't matter because any other package trying to achieve similar functionality would basically do the same. And org prefix doesn't make sense because there is nothing org-specific there.